### PR TITLE
Bot pr arm osx h833286

### PR DESCRIPTION
**Pull request by ericpre**
Fix for #31.

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [n/a] Bumped the build number (if the version is unchanged)
* [n/a] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [n/a] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ericpre** [comment]
Anyone form @conda-forge/scikit-sparse or @conda-forge/core to merge this PR? Thanks!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,13 +8,19 @@ source:
   fn: scikit-sparse-{{ version }}.tar.gz
   url: https://github.com/scikit-sparse/scikit-sparse/archive/v{{ version }}.tar.gz
   sha256: af850a0efc06576c6aa2d8a801316e9a2ae47609e48534b17352aac8b8deeba7
+  patches:
+    - 0001-Fix-import-in-setup.py.patch
```
Comment (jakirkham) [inline recipe/meta.yaml:12]: Could you please send this change upstream as well?

Comment (ericpre) [reply recipe/meta.yaml:12]: This is already done: https://github.com/scikit-sparse/scikit-sparse/pull/84 :)

**Comment by ericpre** [comment]
@jakirkham, is it good to merge? :)

File: recipe/meta.yaml (hunk 1 above)
Comment (hmaarrfk) [inline recipe/meta.yaml:12]:
```suggestion
    # https://github.com/scikit-sparse/scikit-sparse/pull/84
    - 0001-Fix-import-in-setup.py.patch
```

**Comment by ericpre** [comment]
Thanks @hmaarrfk!

**Pull request merged**